.github/PROCESS.md

```diff
@@ -4,8 +4,6 @@ This file contains some process notes for FerretDB Inc. engineers and long-term
 If you are a community contributor, feel free to ignore all this!
 We will do all that for you so you can focus on what is really interesting for you.
 
-We encourage diversity between us as different points of view expand our vision and contribute to the project's success.
```

Comment:
That's still true, but it was out-of-place there. We still have it in our team contract and Code of Conduct